assets/src/dashboard/app/views/templateDetails/header/index.js

```diff
@@ -18,33 +18,78 @@
  */
 import PropTypes from 'prop-types';
 import { __, sprintf } from '@web-stories-wp/i18n';
-
+import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { DetailViewNavBar, Layout } from '../../../../components';
+import {
+  Button,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+  BUTTON_VARIANTS,
+  Icons,
+  themeHelpers,
+} from '../../../../../design-system';
+import { Layout } from '../../../../components';
 import { APP_ROUTES, ROUTE_TITLES } from '../../../../constants';
+import { parentRoute } from '../../../router/route';
+
+const Nav = styled.nav`
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  margin: 48px 0 84px;
+  padding: 0 88px;
+`;
 
-function Header({ onBookmarkClick, onHandleCtaClick }) {
+const HiddenHeading = styled.h2`
+  ${themeHelpers.visuallyHidden};
+`;
+function Header({ onHandleCtaClick, templateTitle }) {
   return (
     <Layout.Fixed>
-      <DetailViewNavBar
-        ctaText={__('Use template', 'web-stories')}
-        closeViewAriaLabel={sprintf(
-          /* translators: %s: page title of link */
-          __('Go to %s', 'web-stories'),
-          ROUTE_TITLES[APP_ROUTES.TEMPLATES_GALLERY]
-        )}
-        handleBookmarkClick={onBookmarkClick}
-        handleCta={onHandleCtaClick}
-      />
+      <Nav>
+        <HiddenHeading>
+          {sprintf(
+            /* translators: %s: template title */
+            __('Template Details for %s', 'web-stories'),
+            templateTitle
+          )}
+        </HiddenHeading>
+        <Button
+          type={BUTTON_TYPES.TERTIARY}
+          variant={BUTTON_VARIANTS.CIRCLE}
+          size={BUTTON_SIZES.SMALL}
+          as="a"
+          aria-label={sprintf(
+            /* translators: %s: page title of link */
+            __('Go to %s', 'web-stories'),
+            ROUTE_TITLES[APP_ROUTES.TEMPLATES_GALLERY]
+          )}
+          href={parentRoute()}
+        >
+          <Icons.Cross />
+        </Button>
```

Comment:
I think this needs to be a little bigger. It's hard to tell from these screenshots but I think the icon itself should be close to 32px, not the bounding button

The button looks to be around 56px: 
<img src="https://user-images.githubusercontent.com/22185279/110032121-048b4580-7cf5-11eb-948e-ebc3f53ba894.png" width="200px" />


|Current|Designs|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/110032033-e9203a80-7cf4-11eb-8521-39ed5c831aca.png)|![image](https://user-images.githubusercontent.com/22185279/110032001-de65a580-7cf4-11eb-8d8c-0afadcf9f460.png)|

Reply:
I swapped in the large cross instead, i think this fixes it.